pennathletics/scraper.py

```diff
@@ -84,22 +91,45 @@ def get_schedule(sport, year):
     :param sport: string value of sport.
     :param year: 4 digitinteger value of year.
     """
-    gameData = []
+    game_data = []
     r = requests.get(
         GAMES_URL.format(sportsdata.SPORTS[sport].SPSID - 1,
                          sportsdata.SPORTS[sport].SPID,
                          year)
     )
     parsed = BeautifulSoup(r.text, "html.parser")
-    info_table = parsed.find_all('table')[0].find_all('tr')
+    info_table = parsed.find_all('table')[1].find_all('tr')
     for row in info_table:
-        data = [row.find_all('td') for td in row][0]
-        parsed = [td.decode_contents(formatter="html").strip(
-        ).replace('&nbsp;', '') for td in data]
-        if len(parsed) > 1:
-            for i in range(0, len(parsed) - 1):
-                print (i, len(parsed))
-                parsed[i] = BeautifulSoup(parsed[i]).decode_contents(
-                    formatter="html").strip()
-                gameData.append(parsed)
-    return gameData
+        data = row.find_all('td')[0:4]
+        schedule = []
+        if len(data) > 1:    
+            for td in data:
+                parsed = td.decode_contents(formatter="html").strip()
+                schedule.append(parsed)
+            schedule[0] = datetime.strptime(schedule[0][5:8] + " " +
+                          schedule[0][9:11] + " " + str(year) + " " +
+                          schedule[3].replace(" ",""), "%b %d %Y %I:%M%p")
```

Comment:
`datetime.strptime` is just beautiful, isn't it?

just kidding I use dateutil or [arrow](http://crsmithdev.com/arrow/) for most project, maybe next time